packages/web/components/QuestChain/UploadProof.tsx

```diff
@@ -70,11 +73,11 @@ export const UploadProof: React.FC<{
   const { imageFile, onResetImage } = dropImageProps;
 
   const onModalClose = useCallback(() => {
```

Comment:
I know I said "especially the textfield persisted" but imho it should persist the files and images too. What you think?

Reply:
sure! :)

Reply:
done!

Reply:
I don't know if it is just me but I am getting mixed results when adding images with drag n drop and clicking doesn't open a file brower. Conversely, clicking the files field opens a file browser and adds the file but dropping a file doesn't work for me.

Reply:
All works for me and the changes i introduced shouldn't have influenced this behaviour.

Reply:
Well, LGTM except for that and I am perhaps a bit of an edge case being on Linux.